deploy/operator.antrea.vmware.com_v1_antreainstall_cr.yaml

```diff
@@ -53,27 +85,34 @@ spec:
     # for the GRE tunnel type.
     #enableIPSecTunnel: false
 
-    # Determines how traffic is encapsulated. It has the following options
-    # encap(default): Inter-node Pod traffic is always encapsulated and Pod to outbound traffic is masqueraded.
-    # noEncap: Inter-node Pod traffic is not encapsulated, but Pod to outbound traffic is masqueraded.
-    #          Underlying network must be capable of supporting Pod traffic across IP subnet.
-    # hybrid: noEncap if worker Nodes on same subnet, otherwise encap.
-    # networkPolicyOnly: Antrea enforces NetworkPolicy only, and utilizes CNI chaining and delegates Pod IPAM and connectivity to primary CNI.
-    #
-    #trafficEncapMode: encap
+    # ClusterIP CIDR range for Services. It's required when AntreaProxy is not enabled, and should be
+    # set to the same value as the one specified by --service-cluster-ip-range for kube-apiserver. When
+    # AntreaProxy is enabled, this parameter is not needed and will be ignored if provided.
+    #serviceCIDR: 10.96.0.0/12
```

Comment:
Question: Is the operator already consuming this parameter to populate Antrea agent config?
Same for serviceCIDRv6 below.

Reply:
Good point, we should set serviceCIDR from cluster network CR, instead of setting in install CR.

Reply:
> Good point, we should set serviceCIDR from cluster network CR, instead of setting in install CR.

Just wonder that how do we deal the the serviceCIDR on kubernetes if we set it from cluster network CR?

Reply:
I‘ll remove this change for this patch. For the case on kubernetes, I'll separate the manifest files for openshift and kubernetes in the patch of 'generic operator'.

Reply:
Approach makes sense. We can only use the network CR on OCP4. For Kubernetes I do not think there is a way to obtain it from any other CR